runner/src/main/scala/stress/utils/Scenarios.scala

```diff
@@ -104,6 +104,23 @@ object Scenarios {
       )
   }
 
+  def bulkOpsScenario(repeats: Int): ScenarioBuilder = {
+    scenario("(Bulk) Ops person")
+      .repeat(repeats)(
+        exec(Releases.getRandomTreeRelease)
```

Comment:
Hmm, not really, the dependency scenario is the only one that uses that call I think... and when running a dependency scenario, you're gonna create a dependency tree?